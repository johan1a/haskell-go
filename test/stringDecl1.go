var x string = "Hello World"
