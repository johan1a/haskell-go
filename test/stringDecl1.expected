[DeclarationStmt (VarDecl [IdDecl "x"] (TypeName (TypNameIdentifier "string")) [StringExpr "Hello World"])]
